### PR TITLE
CoalesceProperties whitespace fix

### PR DESCRIPTION
**Pull request by gadams00**

### CODE REVIEW

File: rewrite-test/src/main/kotlin/org/openrewrite/RecipeTest.kt [hunk 1]
```diff
@@ -70,7 +70,7 @@ interface RecipeTest {
 
         assertThat(result).`as`("The recipe must make changes").isNotNull()
         assertThat(result!!.after).isNotNull()
-        assertThat(result.after!!.printTrimmed(treePrinter ?: TreePrinter.identity<Any>()))
+        assertThat(result.after!!.print(treePrinter ?: TreePrinter.identity<Any>()))
```
Comment (gadams00) [inline rewrite-test/src/main/kotlin/org/openrewrite/RecipeTest.kt:73]: Causes a bunch of rewrite-java test failures, but I think those are actually problems in rewrite-java recipes. Whitespace is relevant, so tests shouldn't use a result tree print result modified by StringUtils.trimIndent, which is done in Tree.printTrimmed but not in Tree.print.

File: rewrite-yaml/src/main/java/org/openrewrite/yaml/CoalesceProperties.java [hunk 2]
```diff
@@ -16,20 +16,29 @@
...
+import org.openrewrite.yaml.search.FindIndentYamlProcessor;
 import org.openrewrite.yaml.tree.Yaml;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class CoalesceProperties extends Recipe {
     public CoalesceProperties() {
-        this.processor = () -> new CoalescePropertiesProcessor<>();
+        this.processor = () -> new org.openrewrite.yaml.CoalesceProperties.CoalescePropertiesProcessor<>();
```
Comment (aegershman) [inline rewrite-yaml/src/main/java/org/openrewrite/yaml/CoalesceProperties.java:27]: Not a critique, just for learning purposes-- are there specific reasons to use the fully qualified name here? Assuming it's for style/clarity in nested-class cases like this, though I'm curious if there's more to it.

**Review by aegershman — APPROVED** [review]
right on. thanks for your time and insight 👍

File: rewrite-yaml/src/main/java/org/openrewrite/yaml/CoalesceProperties.java (hunk 2 above)
Comment (gadams00) [reply rewrite-yaml/src/main/java/org/openrewrite/yaml/CoalesceProperties.java:27]: ooh, that is not a change I meant to make. I probably need to reset my intellij settings.

File: rewrite-java/src/main/java/org/openrewrite/java/format/BlankLinesProcessor.java [hunk 3]
```diff
@@ -46,16 +46,13 @@ public BlankLinesProcessor(BlankLineStyle style) {
                     return c.withSuffix(suffix);
                 }));
             } else {
-                j = minimumLines(j, style.getMinimum().getBeforePackage());
+                j = j.withPrefix(j.getPrefix().withWhitespace(""));
```
Comment (jkschneider) [inline rewrite-java/src/main/java/org/openrewrite/java/format/BlankLinesProcessor.java:49]: The package exists. We need to ensure that it has a minimum number of lines according to the `beforePackage` setting. Setting it to blank doesn't seem right.

**Review by jkschneider — CHANGES_REQUESTED** [review]

File: rewrite-java/src/main/java/org/openrewrite/java/format/BlankLinesProcessor.java (hunk 3 above)
Comment (gadams00) [reply rewrite-java/src/main/java/org/openrewrite/java/format/BlankLinesProcessor.java]: I added a comment to make this a bit more clear, but this turns out to be due to a case where when comments are empty/missing and the package decl is present, a leading newline on the compilation unit gets removed by intellij formatting. There was a unit test to this effect that was failing.

**Comment by gadams00** [comment]
We had some discussion about delegating all the whitespace/newline formatting in various recipes/processors to a direct, scoped call to AutoFormatProcessor, but that turned out to be a bit of an involved change, so we deferring that until a separate issue (https://github.com/openrewrite/rewrite/issues/157)

**Pull request merged**